backend/src/posts.spec.js

```diff
@@ -27,20 +33,30 @@ describe("queries", () => {
                 }
             }
         `;
+
+        let postQuery = () => query({ query: POSTS });
+
+        it("throws error when user is not authorised", async () => {
+            userId = null;
+            await expect(postQuery()).resolves.toMatchObject({
+                data: {
+                    posts: null,
+                },
+                errors: [expect.objectContaining({ message: "Not Authorised!" })],
```

Comment:
I'm gonna steal this! Elegant solution to concisely match the returned object!

Reply:
Thought so too, Robert suggested it in the last PR 👍